tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -499,6 +499,17 @@ def param_value(self, verbose, name, ptype, value, textfile, binfile, command):
                     # special handle the use of Python scripts in Windows environment:
                     # 1. search for a wrapper executable (of the same name) under the same directory with stat.S_IXUSR flag
                     # 2. if no wrapper is present, prepend command with 'python' executable
+                    def split_with_single_quote_enclosed_fixup(command):
+                        in_command = shlex.split(command, posix=False)  # use non-posix method on Windows
+                        if not ("'" in command):
+                            return in_command
+                        new_command = []
+                        for token in in_command:
+                            if token.startswith("'") and token.endswith("'"):
+                                new_command.append("{}".format(token[1:-1]))
```

Comment:
just curious, is `token[1:-1]` different from `"{}".format(token[1:-1])`? since this is only removing the first and the last character, would suggest `token[1:-1]` since it's more straight-forward

Reply:
Good catch. I will remove it.